src/main/java/teammates/ui/controller/InstructorFeedbackCopyAction.java

```diff
@@ -68,18 +70,21 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         // isError == true if an exception occurred above
         
         boolean omitArchived = true;
-        data.instructors = loadCourseInstructorMap(omitArchived);
+        HashMap<String, InstructorAttributes> instructors = loadCourseInstructorMap(omitArchived);
         List<InstructorAttributes> instructorList =
-                new ArrayList<InstructorAttributes>(data.instructors.values());
-        data.courses = loadCoursesList(instructorList);
-        data.existingFeedbackSessions = loadFeedbackSessionsList(instructorList);
+                new ArrayList<InstructorAttributes>(instructors.values());
+        List<CourseAttributes> courses = loadCoursesList(instructorList);
         
-        if (data.existingFeedbackSessions.isEmpty()) {
-            statusToUser.add(Const.StatusMessages.FEEDBACK_SESSION_ADD_DB_INCONSISTENCY);
-        }
+        List<FeedbackSessionAttributes> feedbackSessions = loadFeedbackSessionsList(instructorList);
+        FeedbackSessionAttributes.sortFeedbackSessionsByCreationTimeDescending(feedbackSessions);
+        
+        data.init(courses, null, feedbackSessions, instructors, fs, null, null);
```

Comment:
Should we overload init or write static factories for different versions of init? Different pages were initialising the page data different so there are a few parameters that are almost always null